.expo/README.md

```
@@ -0,0 +1,8 @@
+> Why do I have a folder named ".expo" in my project?
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)